internal/flag/flags.go

```diff
@@ -60,6 +60,21 @@ var (
 		Usage:   "passphrase to open the encrypted private key",
 		EnvVars: []string{"EO_PASSPHRASE"},
 	}
+	BlsPassphraseFlag = &cli.StringFlag{
```

Comment:
Need to add a flag for EcdsaAliasPrivateKey

Reply:
why exactly? it was not required before

Reply:
Before this PR, there was only one password for all keys.
The generate-alias command generated an ECDSA and encrypted it with the same password.
A new flag for the ecdsaAlias private key is needed to avoid invoking the commands with wrong passwords.

Reply:
added